core/include/irods/private/http_api/version.hpp.in

```diff
@@ -5,7 +5,7 @@ namespace irods::http::version
 {
 	// clang-format off
 	inline constexpr const char* binary_name = "@IRODS_HTTP_API_BINARY_NAME@";
-	inline constexpr const char* server_name = "@IRODS_HTTP_API_BINARY_NAME@/@IRODS_HTTP_API_VERSION@ [@IRODS_HTTP_API_GIT_SHA1@]";
+	inline constexpr const char* server_name = "@IRODS_HTTP_API_BINARY_NAME@/@IRODS_HTTP_API_VERSION@ (@IRODS_HTTP_API_GIT_SHA1@)";
```

Comment:
User-Agent info normally uses parenthesis so I went with that. However, according to the following page, it's best not to reveal **too** much information about your server.
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Server

I think including the build SHA is safe for this project because it's tied to the version number (which is public, unless it's behind a proxy).

Including the SHA could help admins and users in case an org decides to run multiple versions of the HTTP API.

Thoughts?

Reply:
I think this is certainly fine to leave for now.  It's helpful.   And will continue to be at least for the near- and mid-term... possibly, forever.

I vote we leave it.